core/templates/pages/contributor-dashboard-page/modal-templates/login-required-modal.component.spec.ts

```diff
@@ -26,18 +26,19 @@ describe('Login Required Modal Content', () => {
   let component: LoginRequiredModalContent;
   let fixture: ComponentFixture<LoginRequiredModalContent>;
 
-  beforeEach(() => {
+  beforeEach(fakeAsync(() => {
```

Comment:
[Optional]If you use waitForAsync here instead of fakeAsync, you can get rid of the .then().